src/coreclr/tools/Common/JitInterface/CorInfoInstructionSet.cs

```diff
@@ -752,6 +752,30 @@ private static InstructionSetFlags ExpandInstructionSetByReverseImplicationHelpe
             return resultflags;
         }
 
+        private static Dictionary<(string, TargetArchitecture), string> AllInstructionSetGroups { get; } = new()
+            {
+                { ("x86-x64",    TargetArchitecture.X64),   "sse2" },
+                { ("x86-x64",    TargetArchitecture.X86),   "sse2" },
+                { ("x86-x64-v2", TargetArchitecture.X64),   "sse4.2 popcnt" },
+                { ("x86-x64-v2", TargetArchitecture.X86),   "sse4.2 popcnt" },
+                { ("x86-x64-v3", TargetArchitecture.X64),   "x86-x64-v2 avx2 bmi bmi2 lznct movbe fma" },
+                { ("x86-x64-v3", TargetArchitecture.X86),   "x86-x64-v2 avx2 bmi bmi2 lznct movbe fma" },
+                { ("skylake",    TargetArchitecture.X64),   "x86-x64-v3 pclmul aes" },
+                { ("skylake",    TargetArchitecture.X86),   "x86-x64-v3 pclmul aes" },
+                { ("armv8.0",    TargetArchitecture.ARM64), "neon" },
+                { ("armv8.1",    TargetArchitecture.ARM64), "armv8 lse crc rdma sha1 sha2 aes" },
+                { ("armv8.2",    TargetArchitecture.ARM64), "armv8.1" },
+                { ("armv8.3",    TargetArchitecture.ARM64), "armv8.2 rcpc" },
+                { ("armv8.4",    TargetArchitecture.ARM64), "armv8.3 dotprod" },
```

Comment:
Any reason we can't match LLVM exactly here: `armv8-a` and `armv8.1-a`?

That is at least consistent and not something "new"

Reply:
I agreed with @jkotas that `-a` looks redundant but no strong opinion here 🤷‍♂️ can change back

Reply:
I don't disagree, but its then a "new concept". There is benefit to us matching what `LLVM` and `GCC` are using since they are "existing prior art" that many devs are already familiar with.

LLVM at least is using `armv8-a` and `armv8.1-a` (`use `.` in the version and always specify `-a`) so it makes sense to provide those as the baseline.